### PR TITLE
Removed broken link in free-programming-books-pt_BR.md

### PR DESCRIPTION
**Pull request by Siddharth-coder13**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo
Fixed broken link for #5229.

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [ ] Search for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists  in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction)

## Followup

- Check the output of Travis-CI for linter errors!

### CODE REVIEW

File: books/free-programming-books-pt_BR.md [hunk 1]
```diff
@@ -332,7 +332,7 @@
 * [Aprenda a Programar](http://www.jmonteiro.com/aprendaaprogramar/)
 * [Conhecendo Ruby - Eustaquio Rangel](https://leanpub.com/conhecendo-ruby/read)
 * [O (comovente) guia de Ruby do Why](http://why.carlosbrando.com)
-* [Ruby on Rails - Desenv. Ágil para Web com Ruby on Rails](http://www.caelum.com.br/apostila-ruby-on-rails/) - Caelum
+* [Ruby on Rails - Desenv. Ágil para Web com Ruby on Rails](https://silo.tips/queue/ruby-on-rails-caelum?&queue_id=-1&v=1608280401&u=MjQwOTo0MDY0OjRkYzM6ZDFkMDo1OTI5OjIxMTQ6NmMwZjo1Y2Qz) - Caelum
```
Comment (eshellman) [inline books/free-programming-books-pt_BR.md:335]: time-limited link

Comment (eshellman) [reply books/free-programming-books-pt_BR.md:335]: In general, we do not link to file sharing sites.

Comment (Siddharth-coder13) [reply books/free-programming-books-pt_BR.md:335]: Ok, I will fix this.

Comment (Siddharth-coder13) [reply books/free-programming-books-pt_BR.md:335]: The problem seems with the sites themselves, maybe they have removed the book. I think we should remove the link then.

**Pull request merged**